osf/migrations/0026_institution_can_request_access.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.2.15 on 2024-12-27 14:08
```

Comment:
Bunch this in with the other migrations on this branch, just to be tidy. Delete both this and 0025 and re-run `makemigrations` and use that file.